src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -359,7 +360,7 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
         internal async Task<IEnumerable<JObject>> GetFunctionTriggers(IEnumerable<FunctionMetadata> functionsMetadata, ScriptJobHostOptions hostOptions)
         {
             var triggers = (await functionsMetadata
-                .Where(f => !f.IsProxy)
+                .Where(f => !(f is ProxyFunctionMetadata))
```

Comment:
Might be good to add an IsProxy extension method to FunctionMetadataExtensions to do this check

Reply:
Yeah makes sense. I did that, then removed it when I added generic codeless, but I am still doing this check enough times to make this a useful extension.